packages/devtools/lib/src/main.dart

```diff
@@ -68,18 +68,19 @@ class PerfToolFramework extends Framework {
           .open('https://github.com/flutter/devtools/issues', '_feedback');
     });
 
-    await serviceManager.serviceAvailable.future;
-    await addScreens();
-    screensReady.complete();
-
-    final settingsScreen = getScreen('settings');
+    final settingsScreen = SettingsScreen();
+    addScreen(settingsScreen);
```

Comment:
instead of doing this, can we just initialize the settings screen like the other screens and just make sure we don't do something odd when the settings icon is clicked on before the app is connected?

Reply:
I did this as you suggested, when settings icon is clicked on before the app is connected we are not doing anything for now.
But since we need to differentiate the settings icon to be beside of the send feedback button, I still need the `showTab` flag in the screen to differentiate which nav should it be appended to, is it fine?

Reply:
sure that is fine.